### ISSUE TITLE
Bridge speed and fan & Bridge flow interaction

### ISSUE THREAD

**Issue by Reconfirefly** [supermerill/SuperSlicer#2340]
### What happened?

What I am trying to achieve: Making SS recognize bridges and apply the bridge speed to them (verifying this with the gcode analyzer, these extrude paths should turn dark blue). I do not want to apply Bridge flow to these, just the speed.

How I am doing this: I have Bridge speed and fan (overhangs_width_speed) set to the default 55%. This should apply the speed settings for any slope (I've included a test object with a 29 degree slope). However, if Bridge flow (overhangs_width) is set to zero, i.e. how one would disable it if they just wanted speed and not flow settings for bridges, the bridge settings are not applied (i.e. SS doesn't recognize any bridges).

![2022-02-09 16_21_14-Window](https://user-images.githubusercontent.com/10100655/153307357-3d17086d-86c8-478e-af15-42c7e297a8fd.png)
![2022-02-09 16_21_54-Window](https://user-images.githubusercontent.com/10100655/153307354-ac567b1b-8414-4cf9-9141-350261c19386.png)

The work-around: Setting overhangs_width to something non-physical like 99% (i.e. only extrusions that are <1% supported) will allow the speed settings to be recognized and applied while ignoring the flow settings.

![2022-02-09 16_21_22-Window](https://user-images.githubusercontent.com/10100655/153307356-54e7fb92-0e36-485d-9f87-90cfc59c387b.png)
![2022-02-09 16_21_35-Window](https://user-images.githubusercontent.com/10100655/153307355-759f8585-9523-4983-bf19-2a2cff3ad1e7.png)

TL;DR - putting in 0 for overhangs_width breaks the overhang recognition.

### Project file & How to reproduce

[bridge_bug.zip](https://github.com/supermerill/SuperSlicer/files/8036795/bridge_bug.zip)

### Version

2.3.57.6

### Operating system

Windows 10

### Printer model

Voron 2.4 AB-B
